Add `clhs.SpatRaster` and `clhs.SpatVector` methods

brownag opened the pull request:
Hey @pierreroudier 

This PR adds simple wrapper methods for SpatRaster and SpatVector object conversion to data.frame and adds {terra} to suggests. Should mostly address https://github.com/pierreroudier/clhs/issues/26

It is currently based off of this PR/branch https://github.com/pierreroudier/clhs/pull/28. If/when that one gets incorporated I will update this PR.

Thanks for your consideration!

brownag committed:
Add `clhs.SpatRaster` and `clhs.SpatVector` methods

pierreroudier commented:
@brownag  With https://github.com/pierreroudier/clhs/pull/28 merged, let me know when you are ready to merge this one :)

brownag committed:
Add tests of terra methods

brownag commented:
@pierreroudier I rebased onto the new master branch after #28 and added some tests for SpatRaster/SpatVector methods. I think this one is good to go if it looks OK to you!

pierreroudier commented:
Thanks @brownag :)
